safe-global/safe-wallet-web
# Refactor: Safe creation hooks + rm unused files

## What it solves

Resolves #2313

## How this PR fixes it

Adds a new created Safe to the Added Safes list as soon as a creation tx is executed.

+ also removed some unused files.

## How to test it

* Create a Safe.
* Right after executing the transaction, close the tab
* Safe should not be added

* Create another Safe
* wait till the tx executes
* check that your new Safe is added
